### PR TITLE
chore(deps): Update Lerna to 6.0.1

### PR DESCRIPTION
**Pull request by SiriusCrain**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -154,13 +155,14 @@
     "husky": "7.0.4",
     "jest": "27.4.5",
     "jest-watch-typeahead": "1.0.0",
-    "lerna": "4.0.0",
+    "lerna": "^6.0.1",
```
Comment (timurhamitov) [inline package.json:158]: Can we set the exact version, without "^"?
And does it actually just works when we jump from v4 to v6?=)

Comment (SiriusCrain) [reply package.json]: We did not use it to its full potential, so there should be no problems, everything is fine locally

**Comment by doniyor2109** [comment]
@SiriusCrain We use Turbo package that is needed for running [tasks](https://github.com/superdispatch/web-ui/blob/531125241d9d1ec5c36a33afaa49879929808666/package.json#L233-L247)

Lerna introduced same concept in [their release](https://lerna.js.org/docs/concepts/task-pipeline-configuration). I wanted to implement this as well.
Lets remove Turbo and replace with Lerna task pipeline.

**Comment by SiriusCrain** [comment]
> 



> @SiriusCrain We use Turborepo package that is needed for running [tasks](https://github.com/superdispatch/web-ui/blob/531125241d9d1ec5c36a33afaa49879929808666/package.json#L233-L247)
> 
> Lerna introduced same concept in [their release](https://lerna.js.org/docs/concepts/task-pipeline-configuration). I wanted to implement this as well. Lets remove Turbo and replace with Lerna task pipeline.

@doniyor2109 done

File: lerna.json [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {
   "version": "0.21.8",
   "npmClient": "yarn",
+  "useNx": true,
```
[the remark this answers is not in the document]
Comment (doniyor2109) [reply lerna.json:4]: You can run `lerna repair` command

https://blog.nrwl.io/lerna-reborn-whats-new-in-v6-10aec6e9091c#7701:~:text=json%20task%20dependencies.-,Lerna%20Repair,-Lerna%20v6%20comes

[the remark this answers is not in the document]
Comment (SiriusCrain) [reply lerna.json]: @doniyor2109 repaired

**Comment by doniyor2109** [comment]
We also need to test publishing
We can test it together

**Comment by doniyor2109** [comment]
@SiriusCrain I will close it temporary. You can reopen it whenever you will have capacity

**Pull request closed without merging**